qfieldcloud_sdk/sdk.py

```diff
@@ -138,6 +241,13 @@ def login(self, username: str, password: str) -> Dict:
         Args:
             username: the username or the email used to register
             password: the password associated with that username
+
+        Returns:
+            Dict: A dictionary containing the authentication token and additional metadata.
```

Comment:
```suggestion
            Dict[str, Any]: Authentication token and additional metadata.
```

Can you please add the key and value types in all occurances for Dict and List for the code related to this PR?